docs/en/setup/backend/configuration-vocabulary.md

```diff
@@ -138,6 +138,7 @@ core|default|role|Option values, `Mixed/Receiver/Aggregator`. **Receiver** mode
 | - | - | database | Database of InfluxDB. | SW_STORAGE_INFLUXDB_DATABASE | skywalking |
 | - | - | actions | The number of actions to collect. | SW_STORAGE_INFLUXDB_ACTIONS | 1000 |
 | - | - | duration | The time to wait at most (milliseconds). | SW_STORAGE_INFLUXDB_DURATION | 1000|
+| - | - | responseFormat | Influxdb response format option. | SW_STORAGE_INFLUXDB_RESPONSE_FORMAT | MSGPACK|
```

Comment:
What options do they have? Could you add a reference link?

Reply:
https://javadoc.io/static/org.influxdb/influxdb-java/2.17/org/influxdb/InfluxDB.ResponseFormat.html

https://github.com/influxdata/influxdb/issues/8330

Reply:
I find some issues may affect the SkyWalking, such as 
> JSON does not support integers above 2^53

SkyWalking has many long fields, which in Java could be 2^63-1

Reply:
@dmsolr told me, there would be issues when using JSON. Same reason as the above number format issue.